sklearn/datasets/_samples_generator.py

```diff
@@ -2121,6 +2121,20 @@ def make_biclusters(
         words using bipartite spectral graph partitioning. In Proceedings
         of the seventh ACM SIGKDD international conference on Knowledge
         discovery and data mining (pp. 269-274). ACM.
+
+    Examples
+    --------
+    >>> from sklearn.datasets import make_checkerboard
```

Comment:
Currently, this example is in `make_biclusters`'s docstring. This needs to be moved to `make_checkerboard`.